principles/fp-019-term-stability.md

```diff
@@ -52,21 +52,15 @@ It is not necessary (and not advisable) to delete the textual definition.
    -  OWL: Use the `term replaced by` annotation property from OMO ([IAO:0100001](http://purl.obolibrary.org/obo/IAO_0100001)) with the value set to the IRI of the relevant term
    -  OBO: Use the `replaced_by:` tag with the value set to the CURIE of the relevant term
 2. Indicate any inexact term replacements:
-   -  OWL: Use the `oboInOwl:consider` annotation property with the value set to the full IRI(s) of the relevant term(s)
-```
-   <oboInOwl:consider rdf:resource="http://purl.obolibrary.org/obo/OBI_0001544">
-``` 
-
-   -  OBO: Use the `consider:` tag with the value set to the CURIE(s) of the relevant term(s)
-```
-   consider: OBI:0001544
-```
+   -  OWL: Use the `oboInOwl:consider` annotation property with the value set to the full IRI(s) of the relevant term(s): `<oboInOwl:consider rdf:resource="http://purl.obolibrary.org/obo/OBI_0001544">`
```

Comment:
I'd like to first try the change I originally made. This change will make it harder to read.